package.json

```diff
@@ -49,7 +49,7 @@
     "@babel/plugin-transform-runtime": "7.1.0",
     "@babel/preset-env": "^7.2.0",
     "@babel/preset-react": "7.0.0",
-    "@material-ui/core": "^4.0.1",
+    "@material-ui/core": "^4.9.2",
```

Comment:
We should test before merging this.

Reply:
Right now, Im working with that version for several month and working fine. I have a fork where Im doing my own changes.